.travis.yml

```diff
@@ -1,9 +1,9 @@
 sudo: false
+dist: trusty
 language: python
 python:
-  - 3.3
-  - 3.4
   - 3.5
+  - 3.6
```

Comment:
The trusty image only has 3.5 by default (although it will use pyenv to install other ones automatically). This drops the older versions which are in security-fix only mode.